assets/pages/bundles-landing/helpers/blackFriday.js

```diff
@@ -45,8 +47,40 @@ function getPaperItems() {
   return [items[0], { heading: 'Save money on the retail price' }, items[1]];
 }
 
+function getPaperItemsForStructureTest() {
```

Comment:
What is different between this function and getPaperItems()? Can we get rid of it?

Reply:
The difference is the return statement and the elements of the items array.

Reply:
* `getPapertItems` returns `return [items[0], { heading: 'Save money on the retail price' }, items[1]];`

* `getPaperItemsForStructureTest` returns `return [items[0], { heading: 'Save money on the retail price' }];`